README.md

```diff
@@ -1 +1,41 @@
 # class-review-backend
+
+# Go Setup
+1. Download and install go
+2. Figure out your GOPATH
+3. cd to $GOPATH/src amd clone this repo there
+4. cd into class-reviews
+5. Install govendor `go get -u github.com/kardianos/govendor`
+6. run 'govendor sync' (You may have to reset terminal on mac if you get govendor command not found)
+
+
+
+# Local Database (MySQL) Setup
+Install MySQL Server
+https://dev.mysql.com/downloads/mysql/
+
+
+(MySQL Workbench or Shell) 
+https://dev.mysql.com/downloads/workbench/
+
+1. Open MySQL Workbench
```

Comment:
These instructions are good, but we have a dev database set up such that everyone doesn't have to set up a local MySQL environment.

Reply:
Feel free to talk to me if you want to update this PR (and also feel free to join #mh-web-team-summer)!